storage/common_flags.go

```diff
@@ -26,3 +26,5 @@ var ArgDbName = flag.String("storage_driver_db", "cadvisor", "database name")
 var ArgDbTable = flag.String("storage_driver_table", "stats", "table name")
 var ArgDbIsSecure = flag.Bool("storage_driver_secure", false, "use secure connection with database")
 var ArgDbBufferDuration = flag.Duration("storage_driver_buffer_duration", 60*time.Second, "Writes in the storage driver will be buffered for this duration, and committed to the non memory backends as a single transaction")
+var ArgKafkaBrokerList = flag.String("storage_driver_kafka_broker_list", "localhost:9092", "kafka broker(s) csv")
+var ArgKafkaTopic = flag.String("storage_driver_kafka_topic", "stats", "kafka topic")
```

Comment:
I don't think these are common flags. They should be in the kafka package (e.g. like [bigquery](https://github.com/google/cadvisor/blob/master/storage/bigquery/client/client.go#L28))